test/index.js

```diff
@@ -23,6 +23,16 @@ describe('@apostrophecms/import-export', function () {
   after(async function() {
     await cleanData([ attachmentPath, exportsPath ]);
     await t.destroy(apos);
+
+    apos = null;
```

Comment:
This block runs once "after" the entire thing right? In that case it is unnecessary to manually reset these properties because these variables fall out of scope when the closure goes away after the last test. Unless I'm missing something?